package/utils/busybox/Config-defaults.in

```diff
@@ -2544,6 +2568,9 @@ config BUSYBOX_DEFAULT_TELNETD
 config BUSYBOX_DEFAULT_FEATURE_TELNETD_STANDALONE
 	bool
 	default n
+config BUSYBOX_DEFAULT_FEATURE_TELNETD_PORT_DEFAULT
+	int
+	default 0
```

Comment:
I would prefer if this is set to port 23 instead of 0. 
0 is also not in range of `BUSYBOX_CONFIG_FEATURE_TELNETD_PORT_DEFAULT`

Reply:
Changed.